protected/views/element/update.php

```diff
@@ -151,83 +147,85 @@
     $('#preview-frame').attr('src', '/element/{$model->id}/preview?' + $('form').serialize());
 });
 JS
-    );
+);
 
 
-    $attributes = [];
-    foreach ($model->colorAttributes() as $attribute) {
-        $attributes[$attribute] = [
-            'type' => Form::INPUT_HTML5,
-            'html5type' => 'color',
-            'allowUnsafe' => true,
-        ];
-    }
+$attributes = [];
+foreach ($model->colorAttributes() as $attribute) {
+    $attributes[$attribute] = [
+        'type' => Form::INPUT_HTML5,
+        'html5type' => 'color',
+        'allowUnsafe' => true,
+    ];
+}
 
-    if (!empty($attributes)) {
-        $form->formConfig['labelSpan'] = 8;
-        echo Form::widget([
-            'form' => $form,
-            'model' => $model,
-            'columns' => 2,
-            'attributes' => $attributes
-        ]);
-    }
-    echo ButtonGroup::widget([
-        'options' => [
-            'class' => [
-                'pull-right',
-                'buttons-row'
-            ],
+if (!empty($attributes)) {
+    $form->formConfig['labelSpan'] = 8;
+    echo Form::widget([
+        'form' => $form,
+        'model' => $model,
+        'columns' => 2,
+        'attributes' => $attributes
+    ]);
+}
+echo ButtonGroup::widget([
+    'options' => [
+        'class' => [
+            'pull-right',
+            'buttons-row'
         ],
-        'buttons' => [
-            Html::submitButton(
-                $this->title,
-                [
-                    'class' => 'btn btn-primary',
-                    'name' => 'action',
-                    'value' => 'refresh'
-                ]
-            ),
-            Html::submitButton(\Yii::t('app', 'Update & go back'), [
+    ],
+    'buttons' => [
+        Html::submitButton(
+            $this->title,
+            [
+                'class' => 'btn btn-primary',
+                'name' => 'action',
+                'value' => 'refresh'
+            ]
+        ),
+        Html::submitButton(
+            $model->isNewRecord
+                ? \Yii::t('app', 'Create element & go back', ['action' => $this->title])
```

Comment:
Use better context, like `button`.
See #775.

Also, test if we can use constants for these (I think it would break the extraction tool, so test locally via CLI)